CHANGELOG

```diff
@@ -1,6 +1,11 @@
 devel
 -----
 
+* Changed default Linux CI compiler to gcc-11.
+
+* Added a fully functional UI for Views that lets users view, modify mutable
+  properties and delete views from the web UI.
+
```

Comment:
The comment seems unrelated to the changes in this PR

Reply:
https://github.com/arangodb/arangodb/commit/b1a216ba4bc3d897815539a8b03cdc9a16fbae3d accidentally deleted 2 CHANGELOG messages. This PR will restore them.